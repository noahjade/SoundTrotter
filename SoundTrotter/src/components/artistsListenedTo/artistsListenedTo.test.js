import renderer from "react-test-renderer";
import { shallow } from "enzyme";
import ArtistsListenedTo from ".";
 
test("Correctly renders default artists listened to component", () => {
    let selectedCountry = {
        code: "NONE",
        name: "NONE",
        artists: []
    }
    const defaultArtistsListenedToComponent = shallow(<ArtistsListenedTo selectedCountry={selectedCountry} />);
    expect(defaultArtistsListenedToComponent).toMatchSnapshot();
});