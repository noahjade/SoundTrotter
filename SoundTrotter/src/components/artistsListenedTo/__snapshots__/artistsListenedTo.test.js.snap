// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correctly renders default artists listened to component 1`] = `
<div
  className="container"
>
  <div>
    <div
      className="title section"
    >
      Artists you've listened to from...
    </div>
  </div>
  <div
    className="artists"
  >
    <div
      className="message"
    >
      Select a country
    </div>
  </div>
</div>
`;
